### DISCUSSION TITLE
Error When I Try with DB_FILTER

### DISCUSSION THREAD

**Discussion by luismata01** [Tecnativa/doodba#412] [Q&A] [answered]
Hi

 When I try to use 
DB_FILTER = "^%d$"
on common.yaml on production server and after use "docker-compose build" I receive this error message
ERROR: Invalid interpolation format for "environment" option in service "odoo": "^%d$"

I need use this for publish another Database on production  instance with subdomain.

 Can you Help me please?

Thank's

Luis Mata

**Comment by joao-p-marques** [comment, marked as answer]
@luismata01 this seems to be happening because `docker-compose` interprets the `$` sign as a variable identifier. You can escape it by changing your variable to `^%d$$`. There is more information in the [official documentation](https://docs.docker.com/compose/compose-file/compose-file-v3/#variable-substitution).
